### PR TITLE
Beach Patrol Text Grammar Fixes

### PR DESCRIPTION
**Pull request by tobes03**
Hey, I've never made a pull request before, so I apologize if I went about this in the wrong way. I edited all the patrols in the beach territory for clarity and accuracy; you can view a general list of changes below. I exclusively edited the actual text of patrols (and left the surrounding code alone), though that sometimes involved adding or removing text inside curly braces.

### Changes made (roughly in order of quantity from most to least):

- **Correcting commas** (mainly in cases of conjunctions or comma splices.)

- **Correcting capitalization/syntax/hyphenation**

- **Correcting verbs and pronouns** (nearly all of the medicine decline texts along the lines of "medicine cat has second thoughts about leaving their patients" didn't have {} for pronouns and verb conjugation. In other text [predominantly in new-leaf medicine cat patrols], {}s were added erroneously when it should be "them." Those are probably a result of overzealous find-and-replacing.)

- **Streamlining redundant phrasing or breaking one long sentence into two**

- **Altering language choices** (such as replacing human turns of phrase, outdated language, or incorrect semantic usage. This bullet point accounts for very few of the changes.)

I erred on the side of _not_ changing things if I wasn't certain or if it was a matter of preference (and not an egregiously wonky sentence). It would be great to chat more with the writing and development team to know what kinds of other changes they would be okay with me making, since there were a lot of places where I was on the fence about fixing or ameliorating something and opted to leave it as-is.

If the team is happy with these changes, I will also go through the other territories' patrols to make corrections and adjustments. I didn't want to put all my eggs in one basket, as it were, because I haven't made a pull request before and didn't want to waste too much time if I made a mistake in how I went about this.

### CODE REVIEW

**Review by Tiritirimatangi — DISMISSED** [review]
New friend new friend excellent new editing friend Tiri thak you so much for your work here excellent excellent excellent

**Comment by Tiritirimatangi** [comment]
Yay yay yaya omg so grateful to have editing help with the patrols!!!

File: resources/dicts/patrols/beach/hunting/newleaf.json
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/hunting/newleaf.json:1]: Looks like you have a failing json test with this file, that means there's likely a trailing comma that you need to delete!  An easy way to find the error is with https://jsonlint.com/

Comment (tobes03) [reply resources/dicts/patrols/beach/hunting/newleaf.json:1]: Thanks so much!! jsonlint did the trick; I must've deleted a comma at the end of another line by mistake. It should pass the test now, unless there's another error I missed?

**Review by Tiritirimatangi — APPROVED** [review]
Looking good!

**Pull request merged**